### PR TITLE
Move arrivals s3 url to locationConstants.js

### LINKED ISSUE #323: URL in actions/index.js should be moved to locationConstants.js
Thanks for making all the necessary front end updates.  Just noting for myself that this URL should live in locationConstants.js, will create an issue for it.

_Originally posted by @exxonvaldez in https://github.com/trynmaps/metrics-mvp/pull/322_

### PR DESCRIPTION
**Pull request by exxonvaldez**
Fixes #323.

## Proposed changes

Maintain consistency with precomputed wait and travel time s3 URLs, which are generated by functions in locationConstants.js.

## Link to demo, if any

n/a

### CODE REVIEW

**Comment by akgupta89** [comment]
conflicts @exxonvaldez

File: frontend/src/locationConstants.js [hunk 1]
```diff
@@ -170,6 +170,20 @@ export function generateWaitTimeURL(dateStr, statPath, timePath) {
   )}/wait-times_${waitTimesVersion}_sf-muni_${dateStr}_${statPath}${timePath}.json.gz`;
 }
 
+const arrivalsVersion = 'v4a';
+
+/**
+ * Generate S3 url for arrivals
+ * @param dateStr - date
+ * @param routeId - route id
+ */
```
Comment (akgupta89) [inline frontend/src/locationConstants.js]: I'd say include a `@return` definition here

File: frontend/src/locationConstants.js [hunk 2]
```diff
@@ -170,6 +170,20 @@ export function generateWaitTimeURL(dateStr, statPath, timePath) {
   )}/wait-times_${waitTimesVersion}_sf-muni_${dateStr}_${statPath}${timePath}.json.gz`;
 }
 
+const arrivalsVersion = 'v4a';
+
+/**
+ * Generate S3 url for arrivals
+ * @param dateStr - date
+ * @param routeId - route id
```
Comment (akgupta89) [inline frontend/src/locationConstants.js]: Looks like you forgot the type of the variables here, it's good IMO to do it since this is JS not TS. 🤷‍♂

Comment (exxonvaldez) [reply frontend/src/locationConstants.js]: Okay, will update.  Was following the pattern established by Joe for the other url methods.

**Comment by exxonvaldez** [comment]
> Looks good! Could you just extract `sf-muni` to a constant so we can more easily port this for other cities?

Sure, there's already an `agencyName` constant in this file.

File: frontend/src/locationConstants.js (hunk 1 above)
Comment (exxonvaldez) [reply frontend/src/locationConstants.js]: Done.

**Pull request merged**